install_dependencies.sh

```diff
@@ -15,7 +15,14 @@ if [ ! -d "odb/odb-2.4.0-x86_64-linux-gnu" ]; then
   wget http://codesynthesis.com/download/odb/2.4/libodb-sqlite-2.4.0.tar.bz2 -O /tmp/libodb-sqlite.tar.bz2
   tar jxvf /tmp/libodb-sqlite.tar.bz2
   cd libodb-sqlite-2.4.0 && ./configure --with-libodb=../libodb-2.4.0 && make && cd ..
+
+  # make easier to find the static libraries
+  cp libodb-2.4.0/odb/.libs/libodb.a .
+  cp libodb-sqlite-2.4.0/odb/sqlite/.libs/libodb-sqlite.a .
```

Comment:
If all these are being copied to the working directory, are all the flags still necessary in the `.travis.yml` file?

> cmake -DBUILD_STATIC=1 -DCMAKE_CXX_COMPILER=${CXX_COMPILER} -DCMAKE_C_COMPILER=${C_COMPILER} -DCMAKE_CXX_FLAGS="-Iodb/libodb-2.4.0 -Iodb/libodb-sqlite-2.4.0" -DCMAKE_EXE_LINKER_FLAGS="-Wl,-rpath,odb/libodb-2.4.0/odb/.libs/ -Lodb/libodb-2.4.0/odb/.libs/ -Wl,-rpath,odb/libodb-sqlite-2.4.0/odb/sqlite/.libs/ -Lodb/libodb-sqlite-2.4.0/odb/sqlite/.libs/"  -DODB_PATH=$PWD/odb/ -G Ninja

Reply:
it's not the working directory, it's the main directory for odb

Reply:
the `ls` command below in this script looks like this in travis:
```
$ ./install_dependencies.sh
skipping odb installation: odb directory found:
libodb-2.4.0 libodb-sqlite-2.4.0 odb-2.4.0-x86_64-linux-gnu
libodb.a libodb-sqlite.a
```

Reply:
but you're right, the flags can be simplified to `cmake -DBUILD_STATIC=1 -DCMAKE_CXX_COMPILER=${CXX_COMPILER} -DCMAKE_C_COMPILER=${C_COMPILER} -DCMAKE_CXX_FLAGS="-Iodb/libodb-2.4.0 -Iodb/libodb-sqlite-2.4.0" -DCMAKE_EXE_LINKER_FLAGS="-Wl,-rpath,odb/ -Lodb/  -DODB_PATH=$PWD/odb/ -G Ninja`

... I think :sweat_smile: